### ISSUE TITLE
slicer 4.11 The DICOM browser cannot pop up when clicking "Load spine CT from DICOM"

### ISSUE THREAD

**Issue by chunshan** [smclach/PedicleScrewSimulator#6]
@lassoan  

When I click "Load spine CT from DICOM", a dialog titled "Database Repair" pops up with content "All the files in the local database are available". In Slicer 4.11, the DICOM browser is not a dialog and cannot pop up?

Thanks!

**Comment by lassoan** [comment]
You can find latest version of this extension (that is compatible with current Slicer versions) here: https://github.com/lassoan/PedicleScrewSimulator

**Comment by chunshan** [comment]
Yes, I use the latest version of the extension from this repo and the 4.11 version. When I click "Load spine CT from DICOM", a dialog titled "Database Repair" pops up with content "All the files in the local database are available". I must switch to the DICOM module and load the wanted DICOM data. And then I switch back to the PedicleScrewSimulator module to continue the following operations.  It is not the same with the 4.10 version.

**Comment by lassoan** [comment]
Go to DICOM module to sort out DICOM database issues before you start the PedicleScrewSimulator. Probably the easiest is to create a new database.

**Comment by lassoan** [comment]
I've added DICOM import and browser buttons to the module now (in the lassoan fork):

![image](https://user-images.githubusercontent.com/307929/105642397-74392580-5e57-11eb-85c0-997b6cb11fa2.png)

**Comment by chunshan** [comment]
Thanks very much for your quick reply!
I downloaded the latest code and try it. It worked. But I selected one series and clicked the "Load" button, the series was loaded but the "DICOM Browser Window" still stayed on the top and hided the view of loaded data. 
It is appreciated if you fix this little problem.

**Comment by lassoan** [comment]
fixed (https://github.com/lassoan/PedicleScrewSimulator/commit/90a81b781fe2d361ae24672e4c49f3809d15789e)
